pimd/pim6_cmd.c

```diff
@@ -1503,6 +1503,20 @@ DEFPY (debug_pimv6_zebra,
 	return CMD_SUCCESS;
 }
 
+DEFUN_NOSH (show_debugging_pimv6,
+	    show_debugging_pimv6_cmd,
+	    "show debugging [pimv6]",
```

Comment:
Shouldn't this be under pim_cmd_common.c?

Reply:
This command is for pimv6, therefore kept it there. The common code is kept in common file.

Reply:
Yep, I had in mind to move both (`show debugging [pim|pimv6]`), but anyway :)